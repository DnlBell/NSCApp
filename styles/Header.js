import { StyleSheet } from 'react-native';
import { Colors, FontSizes, Margins } from'./Common';

export default StyleSheet.create({
    head: {
        flex: 1,
        paddingTop: Margins.DefaultMargin,
        paddingBottom: Margins.DefaultMargin,
        flexDirection:'column',
        alignItems:'center',
        justifyContent:'center',
        backgroundColor: Colors.AccentColorDark
    },
    heroLine1: {
        color: Colors.PrimaryColor,
        fontSize: FontSizes.HeaderFontSize,
    },
    heroLine2: {
        color: Colors.PrimaryColor,
        fontSize: FontSizes.HeaderFontSize,
        fontWeight: 'bold'
    },
    heroTextFrame: {
        flex: 1,
        alignItems:'center',
        justifyContent:'center',
        marginTop: Margins.DefaultMargin,
        marginBottom: Margins.DefaultMargin
    }
});