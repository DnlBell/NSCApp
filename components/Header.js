import React, {Component} from 'react';
import { View, Text, StyleSheet } from 'react-native';
import styles from '../styles/Header'

class Header extends Component {

    render () {
        return(
        <View style={styles.head}>
            <View style={styles.heroTextFrame}>
                <Text style={styles.heroLine1}>Grow your Mind.</Text>
                <Text style={styles.heroLine2}>Expand your World.</Text>
            </View>
        </View>
        )
    }    
}

export default Header;