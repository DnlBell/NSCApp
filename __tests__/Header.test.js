import Header from '../components/Header';
import renderer from 'react-test-renderer';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});