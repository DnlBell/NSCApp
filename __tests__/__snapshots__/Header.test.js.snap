// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5E5E5E",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingBottom": 18,
      "paddingTop": 18,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "marginBottom": 18,
        "marginTop": 18,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "fontSize": 40,
        }
      }
    >
      Grow your Mind.
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "fontSize": 40,
          "fontWeight": "bold",
        }
      }
    >
      Expand your World.
    </Text>
  </View>
</View>
`;
